### ISSUE TITLE
Update training/validation/test set using latest WHO serology assignments

### ISSUE THREAD

**Issue by gbiagini** [lgragert/nn-sero-pytorch#10]

**Comment by gbiagini** [comment]
Serological assignments have been updated using the `rel_dna_ser` file
